DINGO-4393: Calendar screen reader tweaks (#2124)
Skyscanner/backpack: UNRELEASED.md

> Place your changes below this line.

**Added:**
- bpk-svgs:

>   - New `center-location` icon.

Reviewer: I think this probably got added in a rebase?

Author: 🤦

Author: Removed 👍

- bpk-svgs:
  - New `center-location` icon.

**Fixed:**

- bpk-component-calendar:

>   - Calendar Grid column headings and out-of-range dates now have aria-hidden applied.

Reviewer: ```suggestion
  - Calendar Grid column headings and out-of-range dates are now hidden to assistive technology.
```
